.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: go
+os:
+- linux
+- osx
+go:
+- 1.10.x
+- 1.11.x
+- master
+matrix:
+  allow_failures:
+  - go: master
+  fast_finish: true
+go_import_path: github.com/liamg/aminal
+before_install:
+- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sudo apt-get install -y xorg-dev libgl1-mesa-dev
+  gcc-multilib gcc-mingw-w64-x86-64 && go get github.com/mitchellh/gox; fi
+script:
+- make test
+- if [[ $TRAVIS_OS_NAME == 'osx' ]]; then make build-darwin-native-travis; fi
+- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then make build-linux-travis; fi
+- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then make windows-cross-compile-travis; fi
+env:
+  global:
+  - secure: YOUR_SECURE_TOKEN
```

Comment:
I guess I'll just accept this with the placeholders for now, and add my tokens in a later change.

Reply:
it might be the easiest approach to generate tokens to use `travis` cli tool and run `travis setup releases --com` to generate your GitHub tokens and receive encrypted by Travis entries for tokens. In case if you will run Travis not on .COM domain, you don't need `--com` switch as parameter, which basically just adds more access rights for generated GitHub token.